source/backend/apimodels/Models/Concepts/DispositionFile/DispositionFileSaleMap.cs

```diff
@@ -40,8 +42,8 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.SppAmt, src => src.SppAmount)
                 .Map(dest => dest.RemediationAmt, src => src.RemediationAmount)
                 .Map(dest => dest.PimsDispositionPurchasers, src => src.DispositionPurchasers)
-                .Map(dest => dest.PimsDspPurchAgents, src => src.DispositionPurchaserAgents)
-                .Map(dest => dest.PimsDspPurchSolicitors, src => src.DispositionPurchaserSolicitors);
+                .Map(dest => dest.PimsDspPurchAgents, src => src.DispositionPurchaserAgent == null ? null : new List<DispositionSalePurchaserAgentModel> { src.DispositionPurchaserAgent })
+                .Map(dest => dest.PimsDspPurchSolicitors, src => src.DispositionPurchaserSolicitor == null ? null : new List<DispositionSalePurchaserSolicitorModel> { src.DispositionPurchaserSolicitor });
```

Comment:
this looks odd. why would the ef entities be expecting a list but  the model only contain one?

Reply:
Entities per the scaffolding are one to many, (which is not true) the relationship should be 1 to 1.

Reply:
ah, thats not good. If we need db changes does Dough know?